monsterutilities.sh

```diff
@@ -1 +1 @@
-java -jar $(find /home/janek/Projects/monsterutilities/MonsterUtilities*.jar)
\ No newline at end of file
+java -jar $(find MonsterUtilities*.jar)
```

Comment:
Das Skript war nicht wirklich zum publizieren da und ist völlig unnötig, man kann die jar für gewöhnlich auch einfach per doppelklick starten. Ich habe es nur für desktop shortcuts genutzt.

Reply:
Würde in dem Fall einen Eintrag im .gitignore vorschlagen, damit keine absoluten Pfade im Repo landen.